packages/camera/lib/camera.dart

```diff
@@ -227,51 +249,56 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
     try {
       await _channel.invokeMethod(
-        'capture',
+        'takePicture',
         <String, dynamic>{'textureId': _textureId, 'path': path},
       );
     } on PlatformException catch (e) {
+      value = value.copyWith(errorDescription: e.message);
       throw new CameraException(e.code, e.message);
     }
   }
 
-  void _applyStartStop() {
-    if (value.initialized && !_disposed) {
-      if (value.isStarted) {
-        _channel.invokeMethod(
-          'start',
-          <String, dynamic>{'textureId': _textureId},
-        );
-      } else {
-        _channel.invokeMethod(
-          'stop',
-          <String, dynamic>{'textureId': _textureId},
-        );
-      }
+  Future<Null> startVideoRecording(String filePath) async {
+    if (!value.initialized || _disposed) {
+      throw new CameraException(
+        'Uninitialized CameraController',
+        'startVideoRecording was called on uninitialized CameraController',
+      );
+    }
+    try {
+      value = value.copyWith(recordingVideo: true);
+      await _channel.invokeMethod(
+        'startVideoRecording',
+        <String, dynamic>{'textureId': _textureId, 'filePath': filePath},
+      );
+    } on PlatformException catch (e) {
+      value = value.copyWith(errorDescription: e.message);
+      throw new CameraException(e.code, e.message);
     }
   }
 
-  /// Starts the preview.
-  ///
-  /// If called before [initialize] it will take effect just after
-  /// initialization is done.
-  void start() {
-    value = value.copyWith(isStarted: true);
-    _applyStartStop();
-  }
-
-  /// Stops the preview.
```

Comment:
Is there no longer a way to stop the preview without closing the camera?

Reply:
No, in Android you need the preview open at all time (unless you are recording). This seems to be a limitation of the *MediaRecorder*.